Amber Network Entity Submission (#254)
oasislabs/docs: src/operators/running-node-on-amber-network.md

### Initializing an Entity

::: tip NOTE

> If you previously created an entity for The Quest, we ask that you upgrade that

Reviewer: ```suggestion
If you previously created an entity for The Quest, you need to upgrade that
```

#### Initializing Additional Nodes

> <!-- TODO... what do with this section -->

Author: Ya, probably the best option.
